tools/debug-ui/src/CurrentPeersView.tsx

```diff
@@ -58,9 +58,8 @@ function formatTraffic(bytes_received: number, bytes_sent: number): ReactElement
 }
 
 function addDebugPortLink(peer_addr: string): ReactElement {
-    // TODO: use new UI
     return <a
-        href={"http://" + peer_addr.replace(/:.*/, ":3030/debug")}>
+        href={"http://localhost:3000/" + peer_addr.replace(/:.*/, "/")}>
```

Comment:
This should not have the `http://localhost:3000` part since the server can be run anywhere. You can begin the path with / and the browser will use the same host and port as the current page.

Ideally, this should be a NavLink instead to route to the peer page. But that's fine. Not important.